src/strongbox/addon.clj

```diff
@@ -297,7 +305,9 @@
                                   (map strip-trailing-slash)
                                   set)
                 ;; we don't care which game track is used, just that addons are logically grouped.
-                all-addon-data (load-all-installed-addons install-dir :retail)
+                all-addon-data (timbre/with-merged-config {:middleware [(constantly nil)]}
```

Comment:
`;; swallow log output, else warnings for unrelated addons are surfaced for *this* addon`